native/src/modules/common/onLinkPress.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import { Linking } from 'react-native'
+import type { NavigateToInternalLinkParamsType } from '../app/createNavigateToInternalLink'
+import buildConfig from '../app/constants/buildConfig'
+import type { NavigationStackProp } from 'react-navigation-stack'
+
+type NavigateToInternalLinkType = NavigateToInternalLinkParamsType => void
+
+const HIJACK = new RegExp(buildConfig().internalLinksHijackPattern)
+
+const onLinkPress = (url: string,
```

Comment:
This name is too generic in my opinion as it is placed in the root of `common`. Maybe a name like onInternalLinkPress or onAnchorTagPress is better.